gcloud/storage/blob.py

```diff
@@ -358,7 +417,8 @@ def _check_response_error(request, http_response):
             raise make_exception(faux_response, http_response.content,
                                  error_info=request.url)
 
-    def upload_from_file(self, file_obj, rewind=False, size=None,
+    # pylint: disable=too-many-arguments,too-many-locals
```

Comment:
My mistake, I copied this from somewhere else. The actual issue is `too-many-locals`.

Any thoughts on how to resolve that?